Code/src/main/java/nl/utwente/viskell/ui/components/Pair.java

```diff
@@ -0,0 +1,16 @@
+package nl.utwente.viskell.ui.components;
+
+/**
+ * A generic 2-tuple
+ * @param <A> type of the first element, a
+ * @param <B> type of the second element, b
+ */
+public class Pair<A, B> {
```

Comment:
There's also `javafx.util.Pair` if you're dead certain you don't want a proper class for this.